.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,21 @@
 #### Why we need this PR?
-*Pick one*
+*Pick one, remove the rest*
 - It fixes a bug *bug description*
-- It adds new feature needed because *why we need this feature*
+- It adds new feature needed because of *why we need this feature*
 - Refactoring to improve *what*
 
 #### What happened in this PR?
- - Explain solution of the problem, new feature added.
- - What was changed, added, removed?
- - What is most important part that reviewers should focus on?
- - Was this PR tested? How?
- - Were docs and examples updated, if necessary?
+*Fill relevant points, put NA otherwise. Replace anything inside []*
+ - Problem solution in this PR:
+     *[ Explain solution of the problem, a new feature added here. ]*
+ - Affected files and modules:
```

Comment:
I think the previous point was better - this one looks like asking for a list of files, but PR mechanism automates this.

Reply:
Adjusted a bit